jax/_src/pallas/mosaic/interpret.py

```diff
@@ -1197,6 +1200,9 @@ def store(
   val = np.array(val)
   src_device_id = _to_int(src_device_id)
   src_local_core_id = _to_int(src_local_core_id)
+  if output_name is not None:
```

Comment:
For readability of the code - does output_name being set this imply that this came from a BlockSpec? Maybe leave a comment or wrap the 3 arguments together (output_name, block_indices, grid_index) into a single datastructure?